### ISSUE TITLE
Encrypted email field throws an error in email notifications

### ISSUE THREAD

**Issue by joelzerner** [verbb/formie#742]
**Description**

If an email field is set to be encrypted and then used in one of the email-related fields (reply-to, to, cc, bcc, from etc.) in an email notification it will cause an error as per below:

An error occurred.

Notification email parse error for ReplyTo: {field.email}”. Template error: “Address in mailbox given [••••••••••••••••••••••••••••••••] does not comply with RFC 2822, 3.6.2.” /craft/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/Headers/MailboxHeader.php:355

**Additional info**
- Plugin version: 1.5.8
- Craft version: 3.7.30.1
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.8",
  "...": "..."
}
```

Then run `composer update`.

As an aside, how would you expect encrypted fields to show their content is email notifications? It was the case (and the cause of this bug) that it'd be replaced with that content mask, which might be suitable for the email content where you'd want to hide encrypted content, but poses this issue.

Keen to see what others expect with this setting.

**Comment by joelzerner** [comment]
Nice work on the quick fix Josh 👏

Without thinking too hard, the behaviour I'd expect for the "Enable Content Encryption" option would be for it to be stored encrypted in the database but when outputted by Craft, whether in the control panel or in an email notification, it should displayed normally.

**Comment by engram-design** [comment]
@joelzerner Yep, that's what I'm thinking now, and that's the behaviour, thanks!

**Comment by engram-design** [comment]
Fixed in [1.5.9](https://github.com/verbb/formie/releases/tag/1.5.9)
